### ISSUE TITLE
Giving me a warning (incompatible?) with my earphones

### ISSUE THREAD

**Issue by arghanath007** [noisetorch/NoiseTorch#347]
**Describe the bug**
I am using a new wireless earphone and it says with yellow color -> (incompatible?) 

**To Reproduce**
1. Go to 'Turn on bluetooth and connect a bluetooth earphone'
2. Click on 'the noisetorch icon to turn on noisetorch'

**Expected behavior**
It is giving me an incompatible? warming.

**Screenshots**
![image](https://user-images.githubusercontent.com/54589605/178090758-cecf659d-1266-4a18-ab0e-196db90eac6c.png)

**Desktop (please complete the following information):**
 - Ubuntu 20.04
 - GNOME: 
 - Pulseaudio: 
 - e.g. v0.12.0

**Comment by ZyanKLee** [comment]
Hello @arghanath007 ,
The incompatible flag is added to devices that do not report support for dynamic latency.
Your device may work fine, but we can't guarantee it will.

**Comment by arghanath007** [comment]
Ok Got it understood. Thanks for the reply. Any hardware thing or feature for which this is happening or is it just because it is a Bluetooth earphone.

**Comment by TheDukeofErl** [comment]
> Ok Got it understood. Thanks for the reply. Any hardware thing or feature for which this is happening or is it just because it is a Bluetooth earphone.

The issue is likely due to having a version of pulseaudio that is too old. Updating to a newer version of Ubuntu may fix the issue. It doesn't look like you included the version of pulseaudio you're using in the original post but I'm pretty sure 20.04 is on a version that's too old. I believe that 22.04 is new enough to have the required version of pulseaudio for proper functionality. Though this comment is in reference to Linux Mint, https://github.com/noisetorch/NoiseTorch/issues/338#issuecomment-1159818393 may help add some context here.

**Comment by arghanath007** [comment]
Yes I am on Ubuntu 20.04 LTS. 
The pulse audio version is `pulseaudio 13.99.1`
![image](https://user-images.githubusercontent.com/54589605/179158045-9d7fb681-e199-42e4-83bc-7a869350f31a.png)

It it happening with my wireless earphone only. I will try another wireless headphone from my friends and see if this shows or not. So this is happening because of software and not hardware? But my previous wired headphones worked perfectly fine(no incompatibility warning) until I started using this wireless one.

**Comment by ZyanKLee** [comment]
@arghanath007 how did the test with another headphones go? could you find a solution for your headphones?

**Comment by arghanath007** [comment]
Nope. I tried with another wireless headphones but same issue. I think they mentioned the software version is too old. I am using a new wired headphones now and it is working fine for me. Previous wired one also worked fine before I tried the wireless ones. I guess wireless ones have some compatibility issues with the software I guess.
